src/devices/OneWire/OneWireController.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Iot.Device.OneWire
+{
+    public static partial class OneWireController
+    {
+        public static IEnumerable<OneWireBus> EnumerateBuses()
+        {
+            return EnumerateBusesInternal();
+        }
+
+        public static IEnumerable<OneWireDevice> EnumerateDevices(int family = -1)
```

Comment:
Should family be enum instead? I think there is just few 1-wire devices total. How does user know what value to use for say Thermometer?

Reply:
I started with that, but there is no good name for the families. And many families have overlapping feature set, like temp sensors are in 5 families.
[Devices](https://github.com/owfs/owfs-doc/wiki/1Wire-Device-List)
[Families](http://owfs.sourceforge.net/simple_family.html)

Reply:
you can put 2 names for the same value in the enum - i.e.:
```csharp
enum DeviceFamily : byte
{
    Ds18B20 = 5,
    Max31820 = 5,
    DigitalThermometer = 5,
}

```

this way you can have both names but in the implementation use whichever one you have tested. if we ever find any difference we can branch on that

Reply:
Added enum